internal/middlewares/authelia_context.go

```diff
@@ -331,7 +342,7 @@ func (ctx *AutheliaCtx) GetSessionProvider() (provider *session.Session, err err
 // GetCookieDomainSessionProvider returns the session provider for the provided domain.
 func (ctx *AutheliaCtx) GetCookieDomainSessionProvider(domain string) (provider *session.Session, err error) {
 	if domain == "" {
-		return nil, fmt.Errorf("unable to retrieve domain session: %w", err)
+		return nil, fmt.Errorf("unable to retrieve session cookie domain provider: no configured session cookie domain matches the url 'https://auth.notexample.com'")
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to retrieve session cookie domain provider: no configured session cookie domain matches the url '%s'", targetURL)
```

I suspect this was for some testing, also probably points out the fact that we should unit test this if we aren't already.